netmanager/src/views/components/DataDisplay/Devices.js

```diff
@@ -639,23 +649,28 @@ const DevicesTable = (props) => {
             display: 'flex',
             alignItems: 'center',
             justifyContent: 'flex-end'
-          }}>
-          <Button
-            variant="contained"
-            color="primary"
-            type="submit"
-            align="right"
-            onClick={() => setRegisterOpen(true)}>
-            {' '}
-            Add Device
-          </Button>
+          }}
+        >
+          {activeNetwork.net_name === 'airqo' && (
+            <Button
+              variant="contained"
+              color="primary"
+              type="submit"
+              align="right"
+              onClick={() => setRegisterOpen(true)}
+            >
+              {' '}
+              Add Device
+            </Button>
+          )}
           <Button
-            variant="outlined"
+            variant={activeNetwork.net_name === 'airqo' ? 'outlined' : 'contained'}
             color="primary"
             type="submit"
             style={{ marginLeft: '20px' }}
-            onClick={() => setSoftRegisterOpen(true)}>
-            Soft Add Device
+            onClick={() => setSoftRegisterOpen(true)}
+          >
+            {activeNetwork.net_name === 'airqo' ? 'Soft Add Device' : 'Add Device'}
           </Button>
```

Comment:
If the organisation is airqo, the label is shown as soft add device and then for other organisations, it is add device even though the functionality is soft add device